### PR TITLE
Create BinBayyah.net.xml

### PR DESCRIPTION
**Pull request by Giltyhub**

### CODE REVIEW

**Comment by pipboy96** [comment]
```
autoconfig.binbayyah.net
autodiscover.binbayyah.net
cpanel.binbayyah.net
ftp.binbayyah.net
imap.binbayyah.net
localhost.binbayyah.net
mail.binbayyah.net
pop.binbayyah.net
smtp.binbayyah.net
webdisk.binbayyah.net
webmail.binbayyah.net
whm.binbayyah.net
```

**Comment by Giltyhub** [comment]
@pipboy96 thanks added

File: src/chrome/content/rules/BinBayyah.net.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+		ftp.binbayyah.net
+		imap.binbayyah.net
+		mail.binbayyah.net
+		pop.binbayyah.net
+		smtp.binbayyah.net
+		webdisk.binbayyah.net
+		webmail.binbayyah.net
+		whm.binbayyah.net
+-->
+<ruleset name="BinBayyah.net" platform="mixedcontent">
```
Comment (pipboy96) [inline src/chrome/content/rules/BinBayyah.net.xml]: Do all targets have breakage caused by MCB?

Comment (Giltyhub) [reply src/chrome/content/rules/BinBayyah.net.xml]: @pipboy96 yes

**Pull request merged**